Shopkeeper/Shopkeeper.Business/Item.cs

```diff
@@ -12,5 +12,7 @@ public class Item
         public IEnumerable<Item> Components { get; set; }
         public bool HasComponents => Components?.Any() == true;
         public bool RequiresRecipe { get; set; }
+
+        public override string ToString() => $"{Name} <{Id}>";
```

Comment:
It's probably better if I just make a constructor that takes in all the required fields, that way I can ensure they're never null, since if they are null they'd be messing everything up anyway

Reply:
Constructor won't work because the values are brought in via configuration, so they're automatically parsed. I'll add in the null checks